class-elementor.php

```diff
@@ -193,8 +193,8 @@ public function filter_css_file($name, $absolutePath) {
           $baseurl = preg_replace('/https?:\/\//', '', $upload_data['baseurl']);
           $root_dir = trim(ud_get_stateless_media()->get('sm.root_dir'), '/ '); // Remove any forward slash and empty space.
           $root_dir = apply_filters("wp_stateless_handle_root_dir", $root_dir);
-          $root_dir = !empty($root_dir) ? $root_dir . '/' : '';
-          $image_host = ud_get_stateless_media()->get_gs_host() . $root_dir;
+          $root_dir = !empty($root_dir) ? $root_dir : '';
```

Comment:
I keep it here so this prevent $root_dir being `null` or `false` (replacing it to empty string)